common/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2020",
+    "lib": ["es2020"],
+    "outDir": "out",

```

Comment:
Have you tested the debug configuration (breakpoints) and .vsix packaging (install and test)?
I didn't know about npm link, it looks useful.

We should use the same way of sharing code between this PR and my WIP https://github.com/savoirfairelinux/vscode-bitbake/pull/41

Let's figure this out on the next meeting :)

Reply:
You should check:
- [ ]  adding jest __tests__ in the common package
- [ ]  adding a dependency in the common package.json, checking that debugging has it, as well as the .vsix
- [ ]  npm run clean removes all generated files and folders.
- [ ]  generated files are also gitignored
- [ ]  vsix package doesn't contain .ts files
- [ ] breakpoints in the common code get hit when debugging the server and the client
- [ ] vsix works properly once installed